translations/messages.fr.xlf

```diff
@@ -0,0 +1,3061 @@
+        Here, the top left intersection is a suicide for Light.
+        On the bottom right, it would be a suicide for Dark, and on the bottom left it is not a suicide for any player.</source>
+        <target>Au Go le suicide est interdit.
+        Quand mettre une pierre sur une intersection ferait que le groupe de votre dernière pierre n&apos;a aucune liberté et ne capture aucunes pierres, jouer cette intersection serait un suicide, et est donc interdit.
+        Ici, l&apos;intersection en haut à gauche est un suicide pour Clair.
+        En bas à droite, un suicide pour Foncé, et en bas à gauche n&apos;est un suicide pour aucun joueur.</target>
+      </trans-unit>
+      <trans-unit id="2066383177849177665" datatype="html">
+        <source>Life and death (death)</source>
+        <target>Vie et mort (mort)</target>
+      </trans-unit>
+      <trans-unit id="5995146621586456333" datatype="html">
+        <source>From the capture rule follows the life and death notion:
+        dead stones are stones that are definitely capturable (without losing anything else).
+        Alive stones are stones that can never be captured.
+        From the capture rule, Dark can play inside Light&apos;s territory and make a capture.
+        In this case, we say that Light has only one eye (its last freedom) and that Light is dead (even if not yet captured).
+        At the end of the game, the dead stone will count as a capture, and the intersection it occupies as a territory.</source>
+        <target>De la règle de capture découle la notion de vie et de mort :
+        des pierres mortes sont des pierres que l&apos;on est sûr de pouvoir capturer (sans rien y perdre ailleurs).
+        Tandis que des pierres vivantes sont des pierres que l&apos;on ne peut plus espérer capturer.
+        D&apos;après la règle de capture, Foncé peut jouer à l&apos;intérieur du territoire de Clair et le capturer.
+        On dit dans ce cas que Clair n&apos;a qu&apos;un œil (sa dernière liberté) et qu&apos;il est mort (même si pas encore capturé).
+        En fin de partie, la pierre morte est comptée comme une capture, et la case qu&apos;elle occupe comme un territoire.</target>
+      </trans-unit>
+      <trans-unit id="6721138878022657917" datatype="html">
+        <source>Life and death (eyes)</source>
+        <target>Vie et mort (yeux)</target>
+      </trans-unit>
+      <trans-unit id="1084604724991997052" datatype="html">
+        <source>Here, Light cannot play on the top left, nor on the bottom left.
+        Light will never be able to capture Dark.
+        We say that Dark has two eyes (the eye on the top left, and the one on the bottom left) and that Dark is alive.</source>
+        <target>Ici, Clair ne pouvant jouer ni en haut à gauche, ni en bas à gauche, il ne pourra jamais capturer Foncé.
+        On dit alors que Foncé a deux yeux (l&apos;oeil en haut à gauche et celui en bas à gauche) et qu&apos;il est vivant.</target>
```

Comment:
"oeil" au lieu de ... _il ?

Reply:
take care of the others eyes, Basil ! Venge Nicéphore!

Reply:
j'ai remplacé oeil par œil, pas sûr si c'est bien ça le commentaire ?